ci-tooling/test/test_projects_factory.rb

```diff
@@ -41,6 +41,15 @@ def setup
     # fake data in this test which would raise in the adjustment as expections
     # would not be met.
     CI::UpstreamSCM.any_instance.stubs(:releaseme_adjust!).returns(true)
+    stub_request(:get, 'https://projects.kde.org/api/v1/projects/frameworks').
+        with(headers: {'Accept'=>'*/*', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'User-Agent'=>'Ruby'}).
```

Comment:
You can drop the entire with part. if you don't define it everything matches, which is what we want here

Reply:
Fixed.